### DISCUSSION TITLE
Modeling Relay

### DISCUSSION THREAD

**Discussion by Abbas111977** [SpiceSharp/SpiceSharp#187] [Q&A] [answered]
Hi there, 
Can you please help me with modeling D.C relay with 12 V dc source and multiple contacts ( a combination of N.C and N.O contacts ) 
Best Regards

**Comment by svenboulanger** [comment, marked as answer]
I suggest using a voltage- or current-controlled switch to model the relay. This was answered a bit more in-depth on [StackExchange](https://electronics.stackexchange.com/questions/185855/how-can-i-model-a-relay-in-ltspice-iv).

**Reply by Abbas111977** [reply]
Thank you very much for your advice
